ftw/testbrowser/utils.py

```diff
@@ -22,3 +23,16 @@ def verbose_logging():
 
 def parse_html(html):
     return lxml.html.parse(html, TestbrowserHTMLParser())
+
+
+def copy_docs_from_interface(klass):
+    """Class decorator for copying the method documentation from the only
```

Comment:
Thanks for the hint.
I tried another sphinx extension, but it did not work.
The main reason for having an interface is to ensure consistency through all drivers. Therefore it does not appear in the API docs, but the actual drivers do.
I also like that I can inspect the docs in pdb with e.g. `driver.get_url.__doc__`.